### PR TITLE
[wave2-1060] fgs degree cleanup

### PR DESCRIPTION
**Pull request by mhughes2012**

### CODE REVIEW

**Comment by mhughes2012** [comment]
sounds good. I'll wait for it to be merged to fgs.

File: .gitignore [hunk 1]
```diff
@@ -44,6 +44,7 @@ Thumbs.db
 ##########
 .sass-cache
 sites/default/local_database.inc
+*.css.map
```
Comment (robbdavis) [inline .gitignore]: Is that + character supposed to be there?

File: sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.module [hunk 2]
```diff
@@ -74,10 +74,6 @@ function ucgy_fgs_form_alter(&$form, &$form_state, $form_id) {
     $form['field_fgs_general_fees_label']['#disabled'] = true;
     $form['field_fgs_programs_fees_label']['#disabled'] = true;
     $form['field_fgs_total_fees_label']['#disabled'] = true;
-    $form['field_fgs_general_fee_definition']['#disabled'] = true;
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.module]: Looks you have these hidden via CSS, is there a reason the disabled option is being removed for these fields?

File: .gitignore (hunk 1 above)
Comment (mhughes2012) [inline .gitignore]: there's no leading + char in the file. maybe a file compare glitch?

File: sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.module (hunk 2 above)
Comment (mhughes2012) [inline sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.module]: It's not as clear as other tickets but the request is "Make the following fields editable"

File: .gitignore (hunk 1 above)
Comment (mhughes2012) [inline .gitignore]: there's no + char added here, maybe a diff compare file glitch?

File: sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.module (hunk 2 above)
Comment (mhughes2012) [inline sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.module]: It's not formatted as well as other tickets, but the request starts with: "Make the following fields editable"

**Comment by dpfitzsi** [comment]
Looks like there are a lot of conflicts on this one too. Will need to resolve these so I can send to Louise to test.

**Comment by dpfitzsi** [comment]
Mike can you resolve conflicts on this branch?

**Comment by mhughes2012** [comment]
closing in favour of a clean pr in #629

**Pull request closed without merging**